pkg/acsengine/defaults.go

```diff
@@ -624,10 +621,18 @@ func setMasterNetworkDefaults(a *api.Properties, isUpgrade bool) {
 	if a.MasterProfile == nil {
 		return
 	}
-
-	// Set default Distro to Ubuntu
```

Comment:
Unclear why defaulting of the distro is part of `setMasterNetworkDefaults` but this lgtm

Reply:
I think this need to be moved to separate code block. But one change at the time.

Reply:
Agreed it should be moved out in a separate PR